src/session/session_api.c

```diff
@@ -49,6 +49,61 @@ __wt_session_reset_cursors(WT_SESSION_IMPL *session, bool free_buffers)
 	return (ret);
 }
 
+/*
+ * __wt_session_cursor_cache_sweep --
+ *	Sweep the cursor cache.
+ */
+int
+__wt_session_cursor_cache_sweep(WT_SESSION_IMPL *session)
+{
+	WT_CURSOR *cursor, *cursor_tmp;
+	WT_CURSOR_LIST *cached_list;
+	WT_DECL_RET;
+	uint32_t position;
+	int i, t_ret, nbuckets, nchecked, ncleaned;
+	bool caching;
+	bool productive;
```

Comment:
Join the two bool variables onto a single line.

Reply:
Thanks, I've simplified this code, so `caching` is no longer needed.